Frameworks/UIKit/UITableViewCell.mm

```diff
@@ -1482,6 +1482,12 @@ static void removeAllAnimationsFromLayers(CALayer* layer) {
     }
 }
 
+// determine if a UIView is internal subview used for building default chrome for UITableViewCell
+- (BOOL)_isInternalSubview:(UIView*)view {
+    return (view == _contentView || view == _curAccessoryView || view == _groupEdgeView || view == _removeButton ||
+            view == _editSelectedView || view == _backgroundView || view == _selectedbackgroundView || view == _borderView);
```

Comment:
referencing platform expose the same issue. ;/, so it is app issue.

---
In reply to: [94885465](https://github.com/Microsoft/WinObjC/pull/1631#discussion_r94885465) [](ancestors = 94885465)